pyrk/utilities/plotter.py

```diff
@@ -1,10 +1,10 @@
 # Licensed under a 3-clause BSD-style license - see LiCENSE.rst
 import os
 import matplotlib
-matplotlib.use("Agg")
 import matplotlib.pyplot as plt
 import matplotlib.cm as cmx
 import matplotlib.colors as colors
+matplotlib.use("Agg")
```

Comment:
I assume you moved this for a reason -- does something break if you declare the backend first rather than last? (just curious)